pkg/errors/errors.go

```diff
@@ -0,0 +1,12 @@
+package errors
+
+import "github.com/giantswarm/microerror"
+
+var NotFound = &microerror.Error{
```

Comment:
If I don't remember wrong, at one point there was a discussion about grouping errors into reasonable package instead of masking & replicating them from one package to another.

My approach here introduces that and I named the actual error also accordingly so that it's sound when prefixed with `errors`.

Import of this package can be aliased if there's a desire to import stdlib `errors` package and there's problem with package name conflict.

Reply:
I didn't give SIG operator an update yet but I was playing with that and it didn't work. We'd need to move that to a separate repository level. Otherwise we'll always have the same problem but on a repository level instead of package level.

I have a couple suggestions for the SIG and we can partially try them here:

1. Stop using `executionFailedError` and replace `microerror.Maskf(executionFailedError, "text")` with `microerror.Mask(fmt.Errorf("text"))`.
2. Replace `microerror.Maskf(*microerror.Error, ...)` with `microerror.Rootf(error, ...)`
3. Add `microerror.ChRoot(error)` - that would replace root and keep the stack trace.
4. `microerror.Mask` stays at it is.

Then for convenience we would define errors like:

```go
// error.go file

// That's it. No matcher, nothing special.
var NotFound = errors.New("not found")
```

So TL;TR: I'd do it other way around :D No matcher and keep the error bound to the package.

But we can also try to use `errors.New(` instead of `&microerror.Error{` and instead of using `Maskf(` we can use `Mask(fmt.Errorf("%w: text", err))` as long as `Rootf` is not available.

But I won't block you.